app/Command.hs

```diff
@@ -84,6 +84,11 @@ parseCommand s = case runParser command "<interactive>" s of
     sym = L.symbol sc
     sc = L.space space1 (L.skipLineComment "--") (L.skipBlockComment "{-" "-}")
 
+parseBinding :: String -> Maybe Command
```

Comment:
I think that `parseBinding` should return `Either ParseErrorBundle (Binding Term)` in order to report parsing errors to the user